File: API/Entities/Defect.cs
namespace API.Entities
{
    public class Defect : BaseEntity
    {        
        public string Title { get; set; }
        public string DefectDetails { get; set; }
        public string StepsToReproduce { get; set; }
        public string Severity { get; set; }
        public string Priority { get; set; }
        public string DetectedOn { get; set; }
        public string ExpectedResolution { get; set; }
        public string Status { get; set; }
        public string ReportedByTesterId { get; set; }
        public string AssignedToDevloperId { get; set; }        

    }
}

File: client/src/app/shared/models/defect.ts
export interface Defect {
    id: number;
    title: string;
    defectDetails: string;
    stepsToReproduce: string;
    severity: string;
    priority: string;
    detectedOn: string;
    expectedResolution: string;
    reportedByTesterId: string;
    assignedToDevloperId: string;
    status: string;
  }